config/image/oss/kustomization.yaml

```diff
@@ -7,4 +7,4 @@ images:
   newTag: '3.4'
 - name: kic-placeholder
   newName: kong/kubernetes-ingress-controller
-  newTag: '2.12'
+  newTag: '3.0'
```

Comment:
Side note: I'm not sure we still need to bump this version as:
1) we do not distribute `all-in-one-` manifests as an official installation method no more
2) in E2Es we always override it anyway

Reply:
🤫 don't tell but I'm probably still gonna use these for lazy testing; they're quicker for getting a given deploy pattern than setting Helm values for it.